src/github.com/travis-ci/worker/image/api_selector.go

```diff
@@ -0,0 +1,123 @@
+package image
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/cenkalti/backoff"
+)
+
+type APISelector struct {
+	baseURL *url.URL
+}
+
+func NewAPISelector(u *url.URL) *APISelector {
+	return &APISelector{baseURL: u}
+}
+
+func (as *APISelector) Select(params *Params) (string, error) {
+	imageName := "default"
```

Comment:
Looks like this is only used in the `return` at the very end of the function. Can we remove this and just use a literal `"default"` at the end?